### DISCUSSION TITLE
DataObjects not showing up in search. Many-to-One relation type

### DISCUSSION THREAD

**Discussion by blackbender** [pimcore/pimcore#13307] [Q&A / Support] [answered]
Hello everyone!
Objects of the Brand class are not displayed in the search, although everything was done according to the documentation
Can you tell me if I'm wrong somewhere. I don't know what could be the problem.
![image](https://user-images.githubusercontent.com/51505814/193827176-3db00f8a-c6a8-4e10-ae19-6b654ca9a671.png)

**Comment by Maksonis** [comment, marked as answer]
Try to run bin/console pimcore:search-backend-reindex.

**Reply by Anaxarchos** [reply]
@Maksonis I am facing the same problem and that command did not solve it. In my backend the search window is empty for data objects, assets or document links as well.

**Reply by mcop1** [reply]
@Anaxarchos The command should be the solution to your problem. Did you get any errors while executing the command? Can you please check the `search_backend_data` table in your database? Does it contain data sets?

**Reply by Anaxarchos** [reply]
@mcop1 Yes you are right. I didn't get any errors, but the message that the command was processing assets, documents and objects. And I have entries in the search_backend_data table. But I had to click on the refresh button at the bottom of the search window, to get the data displayed. Now everything is working as expected. Thank you.

**Reply by mcop1** [reply]
@Anaxarchos Good to hear :)
Could you please accept Maksonis post as answer? Thank you!

**Comment by Anaxarchos** [comment]
This is already marked as the answer

Am I missing something?

> Am 13.10.2022 um 14:43 schrieb mcop1 ***@***.***>:
> 
> 
> @Anaxarchos <https://github.com/Anaxarchos> Good to hear :)
> Could you please accept Maksonis post as answer? Thank you!
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/pimcore/pimcore/discussions/13307#discussioncomment-3869709>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACIKQQK3RLQSPJE32PCRGVDWC77XTANCNFSM6AAAAAAQ4R64M4>.
> You are receiving this because you were mentioned.
> 

vorneweg.de

Andreas Walter
Woltmershauser Str. 150 
28197 Bremen

Tel: 0170 - 90 28 113
Email: ***@***.***

**Reply by mcop1** [reply]
@Anaxarchos nope, looking good now, sorry, seems like I just did not refresh the page :)
